modules/persistent-storage-csi-drivers-supported.adoc

```diff
@@ -18,7 +18,8 @@ The following table describes the CSI drivers that are installed with {product-t
 
 |AWS EBS (Tech Preview) | ✅ | - | ✅
 |Google Cloud Platform (GCP) persistent disk (PD) (Tech Preview)| ✅ | - | ✅
-|Microsoft Azure Disk (Tech Preview) | ✅ | ✅ | ✅ 
+|Microsoft Azure Disk (Tech Preview) | ✅ | ✅ | ✅
+|Microsoft Azure Stack Hub | - | - | - 
```

Comment:
I did not start the test so I'm not sure if Azure Stack Hub CSI Driver supports the resize/clone/snapshot, I will check it later.
Or @tjungblu, do you have a clear view on this?

Reply:
on ASH I believe it doesn't support any of these features, at least that's what is documented in the limitations from Azure. It's also inline with the end2end test that fail- but by all means: please double check :)

Reply:
Thanks @tjungblu 
@lpettyjo We can keep this, will raise if need update after the test.

Reply:
@tjungblu From last test, it supports resize/clone/snapshot, could you share the azure doc that talk it is not support?

Reply:
@duanwei33 I've been getting this info from https://docs.microsoft.com/en-us/azure-stack/user/azure-stack-managed-disk-considerations?view=azs-2102&tabs=az1%2Caz2 and https://kubernetes-csi.github.io/docs/drivers.html

which I assume are outdated then. But thanks for double checking, much appreciated :)

Reply:
@tjungblu Thanks for the info, I think the CSI overview( https://kubernetes-csi.github.io/docs/drivers.html ) is outdated somehow, from upstream(https://github.com/kubernetes-sigs/azuredisk-csi-driver) and ours(https://github.com/openshift/azure-disk-csi-driver), all are supported on Azure(global), but from the azure doc (https://docs.microsoft.com/en-us/azure-stack/user/azure-stack-managed-disk-considerations?view=azs-2102&tabs=az1%2Caz2), snapshot looks like not be supported although works from my test.
So I'm not sure if snapshot is not production use? I'm thinking have another round test and let's discuss then. 
The resize/clone could be added to doc I think.
cc @lpettyjo

Reply:
> So I'm not sure if snapshot is not production use?

I also wouldn't jump the gun here and advertise something the platform underneath doesn't support. Not sure how this will fit with the GA messaging however. Maybe Duncan can also help you answer what the right approach would be.